src/system/sys_profiler.cc

```diff
@@ -830,15 +830,15 @@ void Profiler::transform_profiler_data (const string &output_file_suffix, const
     PyObject * arguments = PyTuple_New (3);
 
     // set json path location as first argument
-    PyObject * tmp = PyString_FromString (json_filepath.c_str());
+    PyObject * tmp = PyUnicode_FromString (json_filepath.c_str());
```

Comment:
Please, document (python_loader.hh) various string formats used in interaction with python and list corresponding conversion functions provided in PythonLoader or by Python API itself. 

In particular,  here: What is difference between PyUnicode_FromString and PythonLoader::to_py_string?